README.md

```diff
@@ -1,3 +1,98 @@
 # spark-etl
 
-[![Join the chat at https://gitter.im/vngrs/spark-etl](https://badges.gitter.im/vngrs/spark-etl.svg)](https://gitter.im/vngrs/spark-etl?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
\ No newline at end of file
+[![Build Status](https://travis-ci.org/vngrs/spark-etl.svg?branch=master)](https://travis-ci.org/vngrs/spark-etl)
+[![Coverage Status](https://coveralls.io/repos/github/vngrs/spark-etl/badge.svg?branch=master)](https://coveralls.io/github/vngrs/spark-etl?branch=master)
+[![Join the chat at https://gitter.im/vngrs/spark-etl](https://badges.gitter.im/vngrs/spark-etl.svg)](https://gitter.im/vngrs/spark-etl?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+[The ETL(Extract-Transform-Load)] process is a key component of many data management operations, including move data and to transform the data from one format to another. To effectively support these operations, spark-etl is providing a distributed solution.
+
+spark-etl is a Scala-based project and it is developing with Spark. So it is scalable and distributed. spark-etl will process data from N source to N database.
+The project structure:
+- Extract
+  - FILES (json, csv)
+  - SQLs
+  - NoSQLs
+  - Key Value Stores
+  - APIs
+  - Streams
+
+- Transform
+  - Row to json
+  - Row to csv
+  - Json to Row
+  - Change records
+  - Merge records
+
+- Load
```

Comment:
I know it's used as Load in spark but the term "load" leads me to think that we are loading somethin to our system, maybe we should explain it better